packages/amplication-data-service-generator/src/server/resource/service/service.template.ts

```diff
@@ -1,6 +1,5 @@
 import { Injectable } from "@nestjs/common";
-import { PrismaService } from "nestjs-prisma";
-
+import { PrismaService } from "../prisma/prisma.service";
```

Comment:
this code will never be updated in existing apps because this file is ignored. we need to think how to convert this to the user (docs?)

Reply:
I will write a description for the release notes and the docs